homework/04LessonFour.js

```diff
@@ -77,7 +89,7 @@ var nestedObjects = {
   }
 };
 // update the starring property to have the value 'Tim Robbins'
-
+nestedObjects['users'] ['bob roberts'] ['starring'] = 'Tim Robbins';
```

Comment:
Nice. I don't like the spaces in there -> `] [` though

Maybe I'm just easily confused